metaci/build/models.py

```diff
@@ -195,6 +195,7 @@ class Build(models.Model):
         "release.Release", on_delete=models.SET_NULL, null=True, blank=True
     )
     org_note = models.CharField(max_length=255, default="", blank=True, null=True)
+    salesforce_release = models.CharField(max_length=5, blank=True, null=True)
```

Comment:
What do you think about org_salesforce_api_version to try and clarify that we aren't making any statement about what api_version is being used by various parts of the build itself.

Reply:
@prescod according to https://salesforce.stackexchange.com/questions/18905/how-can-i-tell-which-release-my-org-is-running the info from /services/data also includes the release label that people are used to seeing in addition to the api version. The release label would probably be more meaningful to many of our users than the API version. (sorry I'm only bringing this up now after I already merged your CumulusCI pull request; I just found it)

Reply:
Yeah, I had seen that, but I would think it makes more sense to store a computer-readable number instead of a text label in our database. Can you be a bit more explicit about what you would want me to do with the text label?

Unfortunately the 3-digit "release number" does not seem to be provided by the API or any API I'm familiar with. From a performance history point of view, that's actually the thing that's most interesting to me.

Now that we've moved on to the MetaCI half, maybe we should discuss what tricks there are to infer information about the Pod (domain trick, right?), release number(?), service pack number(?), etc. so we can discuss what else we might want to capture.

Reply:
I'm not very excited about maintaining our own mapping of api versions to release labels, but I guess we could.

Reply:
Since 2011, the pattern has never changed (I checked the XML/JSON). So we could extrapolate a decade into the future and only "maintain" it in the unlikely event that Salesforce broke the pattern. On the other hand, I actually don't have a need for the mapping right now, so we could also figure it out when we need that.

Reply:
Also, what about org_salesforce_api_version as a slightly more explicit name?

Reply:
@prescod okay

Reply:
I still want this to be changed to something other than salesforce_release though

Reply:
@prescod my "okay" was for the comment about waiting to add the mapping later. As for org_salesforce_api_version, I think the distinction you're trying to make it between a version specified in a particular API call, vs the highest version that the org's pod supported at the time when the build ran. So maybe pod_api_version rather than org? But org_salesforce_api_version is fine too.